kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -100,6 +100,10 @@ data:
         container_memory_rss_threshold_percentage = 95.0
         # Threshold for container memoryWorkingSet, metric will be sent only when memory working set exceeds or becomes equal to the following percentage
         container_memory_working_set_threshold_percentage = 95.0
+  integrations: |-
+    [integrations.azure_network_policy_manager]
+        collect_basic_metrics = false
```

Comment:
probable adding some comments or links about basic and advance might be helpful customer to refer and understand what metrics will be collected with basic vs advanced.

Reply:
this is tracked by the doc work. Doc is still not ready from npm side.
1) npm doc will have set of metrics collected by them and their dimensions (it will alsopoint to our doc on how to configure collection thru CI)
2) our doc will point to npm doc (#1 above) and also tell how to configure thru this config map.
3) This config map will have links for both 31 & #2 above, once the doc links are available.